CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing to TiVK documentation
+
+We welcome contributions to help improve the TiKV documentation!
+
+The TiKV docs are written in Markdown and use the [Google Developer Documentation Style Guide](https://developers.google.com/style/). Don't worry if this is new to you, we are happy to guide you along the way.
+
+Before contributing, make sure that:
+
+- You have logged into your GitHub account.
+- You are familiar with the Markdown language used by the documentation.
+ 
+## How to contribute to Docs
+
+There are a couple of ways you can contribute to TiKV documentation.
+
+### Review open PRs
+
+Even though you might not be assigned as a reviewer, you are welcomed to check our [open pull requests](https://github.com/tikv/website/pulls) (PRs) and provide your reviews for the ones that interest you. Leave your comments and we can take it from there.
+
+### Propose a change
+
+If you spot any issue in the documentation that requires a change, and you feel like you are not ready to do it, feel free to propose a change by [creating an issue](https://github.com/tikv/website/issues). A good issue help us locate the problem quickly and come up with the fix.
+
+### Submit your Own PR
+
+If you have a specific idea of a fix or update, and you have everything ready (mostly Git environment), follow the steps below to submit a PR:
+
+1. Fork the repository and clone it to your local:
```

Comment:
```suggestion
1. **Fork the repository** then clone it to your local machine:
```

Users might not realize `clone` doesn't fork.

Reply:
Not sure that I follow the edit here. Why would `clone` not work?

Reply:
If the user did not hit the fork button yet the target repository would not exist which would mean they'd get an error